crates/cheatnet/tests/common/scarb.rs

```diff
@@ -0,0 +1,80 @@
+use anyhow::{anyhow, Context, Result};
+use camino::Utf8PathBuf;
+use cheatnet::cheatcodes::ContractArtifacts;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::fs;
+
+// TODO(#41) TAKEN FROM FORGE, REMOVE AFTER SCARB HAS BEEN MOVED TO A SEPARATE PACKAGE
```

Comment:
Can we plan to do this soon? Please remind me at planning so that we don't have to copy code like that 😬

Reply:
I presume that circular imports would be the problem, right?

Reply:
Yeah I've planned to to this after this giant chain of test moving PRs

Reply:
Isn't it possible to use this code from `forge` crate instead of copying it?